tests/integration/test_issues.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+
+def test_issues_export(project):
+    exported_issues_url = project.export_issues()
+    assert exported_issues_url is not None
+    assert exported_issues_url is not ''
+
+    exported_issues_url = project.export_issues("Open")
+    assert exported_issues_url is not None
+    assert exported_issues_url is not ''
+    assert "?status=Open" in exported_issues_url
+
+    exported_issues_url = project.export_issues("Resolved")
+    assert exported_issues_url is not None
+    assert exported_issues_url is not ''
+    assert "?status=Resolved" in exported_issues_url
```

Comment:
Each of these groups of two could be squashed into:
```
assert exported_issues_url
```
Since both None and '' are falsy